src/test/scala/cognite/spark/v1/wdl/WDLRDDCursorLimitTest.scala

```diff
@@ -0,0 +1,75 @@
+package cognite.spark.v1.wdl
+
+import cognite.spark.v1.{DataFrameMatcher, SparkTest}
+import org.apache.spark.sql.internal.SQLConf
+import org.scalatest.{BeforeAndAfterAll, FunSpec, Inspectors}
+
+class WDLRDDCursorLimitTest
+    extends FunSpec
+    with SparkTest
+    with Inspectors
+    with DataFrameMatcher
+    with BeforeAndAfterAll {
+
+  import RowEquality._
+
+  private val sparkReader = spark.read
+    .format("cognite.spark.v1")
+    .option("project", "jetfiretest2")
+    .option("type", "welldatalayer")
+    .useOIDCWrite
+
+  val testClient = new TestWdlClient(writeClient)
+
+  private val testNptIngestionsDF = spark.read
+    .schema(testClient.getSchema("NptIngestion"))
+    .json("src/test/resources/wdl-test-ntp-ingestions.jsonl")
+
+  private val testNptsDF = spark.read
+    .schema(testClient.getSchema("Npt"))
+    .json("src/test/resources/wdl-test-ntps.jsonl")
+
+  override protected def beforeAll(): Unit = {
+    SQLConf.get.setConfString("spark.sql.legacy.respectNullabilityInTextDatasetConversion", "true")
```

Comment:
Transformation runs with common Spark configs, do you need this change only for testing or also later when deploying Transformation with WDL, this can be breaking change and create issues for existed transformations 😅

Reply:
In this exact place it is only for testing.
But I believe this SQLConf is specific for Spark Session here and session is local to SparkTest parent of WDLRDDCursorLimitTest. So this config instance is specific to this test suite.

Reply:
I've removed it now. In some cases, it added better error messages, but it isn't necessary and adds confusion imho.